src/components/TableList/TableList.js

```diff
@@ -155,6 +155,17 @@ function generateTableHeaders(label) {
   ];
 }
 
+function renderNoDataMessage(modelData) {
+  if (modelData.length === 0) {
+    return (
+      <div className="table-list_no-models-message">
```

Comment:
We dont' really have a design for this but I imagine it'll take on some of the styling from the overall design yet to come.

Reply:
Double underscore BEM notation